src/orion/core/io/database/__init__.py

```diff
@@ -264,3 +265,7 @@ class Database(AbstractDB, metaclass=SingletonFactory):
     """
 
     pass
+
+
+# set per-module log level
+logging.getLogger('filelock').setLevel('ERROR')
```

Comment:
Wait, ERROR? Isn't that always going to log? Maybe we should add a unit-test for this logging. :)

Reply:
it's the opposite actually..

given we cannot change the level in the module `filelock`, which is info, we change the required level for logging to error.

so, now, when the code runs on the line `logger.info`, info < error, and it does not print.

Reply:
Oh yes, my bad! I got confused. :sweat_smile: